tests/backend/topology/test_random.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Import modules
+import pytest
+import numpy as np
+
+# Import from package
+from pyswarms.backend.topology import Random
+
```

Comment:
Please add a test for `__compute_neighbors` :+1: 
This seems to be an integral part of the topology and it would be nice if we can make sure that it will always work as expected. :100: 

You can construct a "smaller" swarm in `conftest.py` to make it more manageable.

Reply:
What do you think is the best way to test it? The elements of the output matrix are random so we can't check if it returns the "right" elements. The only things we could test are the shape and the symmetry. We could use small swarms but then it would just be lucky if we get the right answers or not.

Reply:
Implemented a test for the shape and the symmetry, I hope it is sufficient for you 🙈.

Reply:
I really wonder why my earlier reviews didn't show up :confused: So you can probably try to set a random seed (not sure if we set the numpy random seed or the standard library's `random` seed) to some value so that it will always return the correct values whenever the `__compute_neighbors()` is called.

Reply:
So yes, just check if setting a random seed will work so that we always have an expected behaviour